src/main/java/com/mineboundteam/minebound/registry/BlockRegistry.java

```diff
@@ -25,6 +25,7 @@ public class BlockRegistry {
     public static final RegistryObject<Block> MYRIAL_CONSOLE = BLOCKS.register("myrial_console", () -> new MyrialConsoleBlock(Properties.copy(Blocks.NETHERITE_BLOCK).lightLevel((i) -> 8)));
     public static final RegistryObject<Block> MYRIAL_GLASS = BLOCKS.register("myrial_glass", () -> new Block(Properties.copy(Blocks.GLASS).noOcclusion().requiresCorrectToolForDrops().strength(30f, 500f)));
 
+    public static final RegistryObject<Block> MYRIAL_STEEL_BLOCK = BLOCKS.register("myrial_steel_block", () -> new Block(Properties.copy(Blocks.NETHERITE_BLOCK)));
```

Comment:
For the sake of consistency and style, you should remove the empty line before the `MYRIAL_STEEL_BLOCK` declaration and add an empty line below.

Reply:
Think I fixed what you meant